homework-3/script.js

```diff
@@ -0,0 +1,154 @@
+let dataArray = [
+  16, -37, 54, -4, 72, -56, 47, 4, -16, 25, -37, 46, 4, -51, 27, -63, 4, -54,
+  76, -4, 12, -35, 4, 47,
+];
+
+// Given array [16,-37,54,-4,72,-56,47,4,-16,25,-37,46,4,-51,27,-63,4,-54,76,- 4,12,-35,4,47] Find the sum and number of positive elements
+
+function sumAndQuantityPositiveElements(numbers) {
+  let sumOfdataArray = 0;
+  let counter = 0;
+  for (let i = 0; i < dataArray.length; i++) {
+    if (numbers[i] > 0) {
+      sumOfdataArray += numbers[i];
+      counter++;
+    }
+  }
+  return [sumOfdataArray, counter];
+}
+console.log(sumAndQuantityPositiveElements(dataArray));
+
+// Find the minimum array element and its sequence number
+
+function minimumArrayValue(numbers) {
+  let minimumValue = numbers[0];
+  let indexMinimumValueVariable;
+  for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] < minimumValue) {
+      minimumValue = numbers[i];
+    }
+  }
+  for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] === minimumValue) {
+      indexMinimumValueVariable = i;
+    }
+  }
```

Comment:
remember DRY principle of programming. You have 2 loops with the same conditions. So maybe it's better to place both logic in one loop?